### DISCUSSION TITLE
FiveM (GTA V)

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#16] [Game request]
General info: https://fivem.net/

Server guide: https://docs.fivem.net/docs/server-manual/setting-up-a-server/

**Comment by RootzTriiCkz** [comment]
Yes please!

**Comment by mvaessen** [comment]
FiveM ended as being the top requested game in the recent poll. So i'm converting it to a development item. See #466

**Reply by mvaessen** [reply]
Unfortunately due to issues with FiveM not wanting to work inside a docker container nicely, this has been postponed until further notice

**Reply by LegitGammin** [reply]
Any updates?

**Reply by mvaessen** [reply]
Hi, no updates on the gsa implementation for this game. Have been working on other gsa things 😄

But, next in line is the new custom containers feature. And that update will let you run any custom docker image via dediconnect.

Including official LinuxGSM support (they recently gave me ✅), maybe even for Windows depending on how WSL works. Haven't looked into that much yet.

Hopefully we have improved conditions for fivem and other games for both Windows and Linux in roughly a month (or 3) 🤞

**Reply by LegitGammin** [reply]
Awesome, thanks for the update!

**Comment by mvaessen** [comment]
A blueprint was made for FiveM / RedM. You should now be able to spin up a game server with a blueprint
https://discord.com/channels/315049169071112193/1253728092040728720
